src/main/java/org/jenkinsci/plugins/ghprb/GhprbRootAction.java

```diff
@@ -86,13 +87,15 @@ public void doIndex(StaplerRequest req, StaplerResponse resp) {
 
                 for (GhprbRepository repo : getRepos(issueComment.getRepository())) {
                     logger.log(Level.INFO, "Checking issue comment ''{0}'' for repo {1}", new Object[] {issueComment.getComment(), repo.getName()});
-                    repo.onIssueCommentHook(issueComment);
+                    if(repo.checkSignature(body, signature))
```

Comment:
You mean to change it for something like this?
```
repo.getTrigger().checkSignature(body, signature)
```

That would basically be the same... or am I missing something?